math/gnuplot/Portfile

```diff
@@ -126,7 +126,7 @@ variant old_bitmap_terminals description "Enable PBM (Portable Bit Map) and othe
     configure.args-replace      --without-bitmap-terminals --with-bitmap-terminals
 }
 
-default_variants                +aquaterm +luaterm +pangocairo +wxwidgets +x11
+default_variants                +luaterm +pangocairo +qt5
```

Comment:
We can discuss removal of `aquaterm` and `x11` from default variants, I'm not against that.

But I feel that `wxWidgets` still works sufficiently well that the requirement to always install the `+qt5` bloatware (which doesn't even work on older macs, so the port would be broken there) is hard to justify.

Reply:
My idea born from three observations:
- the easy way to save the graph during visualization to multiple format (ex. pdf/png/svg)
- many Linux distribution is using Qt as default and the upstream seems to pay more attention to the Qt code (from gnuplot documentation `It is now the fastest and most full-featured interactive terminal option.`)
- the fact that nowadays many programs already use Qt5 and therefore it is already installed (Octave is one example).

Maybe we can choose defaults based on OS version though I don't like too much this idea.

Reply:
updated the PR